dev/snippets/nav/bar_basic.cljs

```diff
@@ -1,11 +1,14 @@
 #_
 (:require [om-bootstrap.button :as b]
-          [om-bootstrap.nav :as n])
+          [om-bootstrap.nav :as n]
+          [om-tools.dom :as d :include-macros true])
 
 (n/navbar
- {}
+ {:brand (d/a {:href "#"}
+              "Navbar")
+  :toggle-nav-key 0}
  (n/nav
-  {:collapsible? true}
+  {:key 0}
```

Comment:
now I can click lines too! Why remove this option? Also, what does adding a key here do? In React you add a Key to force React not to reuse the component with fresh data. This is the only nav, so I don't understand the purpose, or what it adds to the example.